src/utils/_variables.scss

```diff
@@ -0,0 +1,16 @@
+$day-size: 100px;
+$day-count: 31;
+$border-size : $day-size / 100;
+$gap-size: $day-size / 100;
+$mockup-padding: $day-size / 10;
+$calendar-height: 100vh;
+$mockup-size: ($gap-size * 6) + (7 * $day-size);
```

Comment:
($gap-size * 6)
but
(7 * $day-size)

why the order is not the same?
It is confusing

Reply:
Good suggestion ! There is a possbility to improve redability of code and reduce possbility of human error